sklearn/datasets/_base.py

```diff
@@ -340,7 +340,8 @@ def load_csv_data(
         Description of the dataset (the content of `descr_file_name`).
         Only returned if `descr_file_name` is not None.
```

Comment:
```suggestion
        Only returned if `descr_file_name` is not None.

    encoding : str, optional
        Text encoding of the CSV file.

        .. versionadded:: 1.4
```

Reply:
I cannot make the suggestion to acutally add `encoding="utf-8"` as extra kwarg to the prototype of `load_csv_data` above because it is collapsed in the github diff view.